evap/staff/views.py

```diff
@@ -442,7 +442,7 @@ def semester_participation_export(request, semester_id):
         number_of_required_courses_voted_for = semester.course_set.filter(voters=participant, is_rewarded=True).count()
         number_of_optional_courses = semester.course_set.filter(participants=participant, is_rewarded=False).count()
         number_of_optional_courses_voted_for = semester.course_set.filter(voters=participant, is_rewarded=False).count()
-        earned_reward_points = RewardPointGranting.objects.filter(semester=semester, user_profile=participant).exists()
+        earned_reward_points = RewardPointGranting.objects.filter(semester=semester, user_profile=participant).aggregate(Sum('value'))['value__sum']
```

Comment:
this is currently not consistent with the other columns which show a `0` if there are no courses. can you please add an `or 0` so that also in this column every line has an entry?